resources/service_sysvinit.rb

```diff
@@ -11,6 +11,10 @@
   node['platform_version'].to_i == 7
 end
 
+provides :push_jobs_service, platform: 'suse' do |node|
+  node['platform_version'].to_f <= 11.4
```

Comment:
You're probably better off just saying less than 12 here in case for some reason we end up with an 11.5 in the future

Reply:
agreed, I have updated with the last commit